src/main/java/uk/gov/hmcts/reform/civil/service/CoreCaseDataService.java

```diff
@@ -153,4 +156,14 @@ public LocalDate getAgreedDeadlineResponseDate(Long caseId, String authorization
         }
         return null;
     }
+
+    public SearchResult getAllCases(String authorization) {
```

Comment:
perhaps the method should not be all cases as it returns 1000 cases. perhaps it could have a different name. What do you think? as it would be misleading

Reply:
changed the name.